src/prx/main.py

```diff
@@ -483,7 +489,7 @@ def signal_2_carrier_frequency(row):
             flat_obs.loc[
                 mask,
                 "iono_delay_m",
-            ] = 0
+            ] = np.nan
```

Comment:
@plutonheaven not 100% sure here - do we want 0 or NaN for elements we cannot compute?

Reply:
Ok, let's put a NaN. Users will always be able to replace this by 0 if they want to.